src/components/utils.js

```diff
@@ -0,0 +1,33 @@
+export const createReviewCards = (reviewData) => {
+	return reviewData.map((review) => {
+		return (
```

Comment:
you could have all this returned element as a "ReviewCard" and then map through and just create review cards passing a review in props